src/Tribe/Freemius.php

```diff
@@ -15,8 +15,9 @@ class Tribe__Freemius {
 	private $instances = [];
 
 	/**
-	 * Loading of the vendor files for Freemius vendor
-	 * Freemius class should only be loaded once since it will be registred as a Singleton
+	 * Load the vendor files for Freemius vendor.
+	 *
+	 * Freemius class should only be loaded once since it will be registered as a Singleton.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 92 characters

<sub>hash: 50ab26b88612040d6c22390d12d4ca93</sub>